src/frontend/src/lib/utils/token.utils.ts

```diff
@@ -19,17 +22,28 @@ import { nonNullish } from '@dfinity/utils';
  * @returns {number} The maximum amount for the transaction.
  */
 export const getMaxTransactionAmount = ({
```

Comment:
the docstring should be changed too right? from `bigint` to `BigNumber`

Reply:
Good catch, updated.